core/graph/src/main/java/uk/gov/gchq/gaffer/operation/export/graph/ExportToOtherAuthorisedGraph.java

```diff
@@ -88,6 +88,16 @@ public void setKey(final String key) {
         return (TypeReference) new TypeReferenceImpl.Object();
     }
 
+    @Override
+    public Operation shallowClone() {
```

Comment:
Please can you update the return type to be ExportToOtherAuthorisedGraph. Similarly with the other operations.